runtime/jvmti/jvmtiHook.c

```diff
@@ -381,8 +383,11 @@ processEvent(J9JVMTIEnv* j9env, jint event, J9HookRedirectorFunction redirectorF
 			return redirectorFunction(vmHook, J9HOOK_VM_INITIALIZED, jvmtiHookVMInitialized, OMR_GET_CALLSITE(), j9env);
 
 		case JVMTI_EVENT_VM_START:
-			return redirectorFunction(vmHook, J9HOOK_VM_STARTED, jvmtiHookVMStarted, OMR_GET_CALLSITE(), j9env) &&
+			return redirectorFunction(vmHook, J9HOOK_VM_STARTED, jvmtiHookVMStarted, OMR_GET_CALLSITE(), j9env)
+#if JAVA_SPEC_VERSION < 9
+					&&
 					redirectorFunction(vmHook, J9HOOK_JAVA_BASE_LOADED, jvmtiHookModuleSystemStarted, OMR_GET_CALLSITE(), j9env);
```

Comment:
The semi-colon which ends this `return` statement must be outside the conditional else it won't compile for Java 8.
Also, I'd prefer the `&&` and the function call on the same line.